example/src/pages/App.tsx

```diff
@@ -283,41 +305,26 @@ class App extends React.Component<any, any> {
   public getNetwork = () => getChainData(this.state.chainId).network;
 
   public getProviderOptions = () => {
-    const infuraId = "e8ceeaaa4eaa447fa137b1b2f8b6b0a2";
+    const infuraId = process.env.REACT_APP_INFURA_API;
+    const wallyClientId = process.env.REACT_APP_WALLY_CLIENT_ID;
 
     const providerOptions = {
-      walletconnect: {
-        package: WalletConnect,
-        options: {
-          infuraId,
-        },
-      },
       coinbasewallet: {
         package: CoinbaseWalletSDK,
         options: {
           appName: "Web3Modal Example App",
-          infuraId,
-        },
+          infuraId
+        }
       },
-
-      // wallyconnect: {
-      //   package: WallyConnector, // required
-      //   options: {
-      //     clientId: "103be027-a1a6-486c-ae24-0d19909b36d4", // required
-      //     isDevelopment: false,
-      //     devUrl: "http://localhost:3000", // optional
-      //   },
-      // },
-
       "custom-wallyconnect": {
         display: {
           logo: wallylogo,
           name: "WallyConnect",
-          description: "Connect to Wally",
+          description: "Connect to Wally"
         },
         options: {
-          clientId: "15672a04-5ce6-48ff-991c-54ab200bdd5b", // required
-          didHandleRedirect: true,
+          clientId: wallyClientId, // required
+          didHandleRedirect: true
```

Comment:
hmmm where are these options getting used? i'm not sure why you would need `didHandleRedirect`